cudapoa/src/cudapoa_kernels.cuh

```diff
@@ -30,7 +29,8 @@
 #include <claraparabricks/genomeworks/cudapoa/batch.hpp>
 
 // considering 65536 register file size, __launch_bounds__(1024) translates to max 64 register usage
-#define GW_POA_KERNELS_MAX_THREADS_PER_BLOCK 1024
+#define MAX_THREADS_PER_BLOCK_64 1024
```

Comment:
for macros it's good to add a prefix to the name to clarify the scope of usage. hence the `GW_POA_KERNELS` prefix. let's add it (or some version of that) back. and also add `REGISTERS` as a suffix (`_64_REGISTERS`)

Reply:
seems a branch of optimization efforts was applied to minimize the length of this macro! :D changed back and added REGISTERS to be more clear.